### PR TITLE
Add support for writing batches of spans at a time.

### PR DESCRIPTION
**Pull request by csdigi**
This API allows clients to write a higher number of spans without saturating the network.

### CODE REVIEW

File: zipkin/scribe_writer.py [hunk 1]
```diff
@@ -13,14 +13,20 @@ def __init__(self, host, port,  category='zipkin'):
...
         data = self.__build_log_message(span)
         message = "\\n".join([r for r in data.split("\n") if r != ""]) + "\n"
+        return Scribe.LogEntry(category=self.category, message=message)
+
+    def write_multiple(self, spans):
+        entries = [self.__span_to_entry(span) for span in spans]
         self.transport.open()
-        entry = Scribe.LogEntry(category=self.category, message=message)
-        self.client.Log([entry])
+        self.client.Log(entries)
```
Comment (fsiddiqia) [inline zipkin/scribe_writer.py:24]: sweet, i thought you'd actually have to modify the client to accept multiple messages

LGTM

File: zipkin/scribe_writer.py [hunk 2]
```diff
@@ -13,14 +13,20 @@ def __init__(self, host, port,  category='zipkin'):
         self.client = Scribe.Client(iprot=protocol, oprot=protocol)
         self.category = category
 
-    def write(self, span):
+    def __span_to_entry(self, span):
```
Comment (fsiddiqia) [inline zipkin/scribe_writer.py:16]: is the double `_` intentional?

**Comment by peterebden** [comment]
LGTM also, thanks Will!

File: zipkin/scribe_writer.py (hunk 2 above)
Comment (csdigi) [inline zipkin/scribe_writer.py:16]: Follows the style that is already there, I would have used a single _ if I had the choice :)

**Pull request merged**